includes/namespace.php

```diff
@@ -29,64 +29,148 @@
+
+/**
+ * Displays an admin notice about why the plugin is unable to load.
+ *
+ * @since 1.0.0
+ *
+ * @return void
+ */
+function _print_missing_build_admin_notice() {
+	global $web_stories_compatibility;
+
+	$web_stories_compatibility->run_checks();
+	$_error = $web_stories_compatibility->get_error();
+	if ( ! $_error->errors ) {
+		return;
 	}
 
-	add_action( 'admin_notices', __NAMESPACE__ . '\_print_missing_build_admin_notice' );
+	?>
+	<div class="notice notice-error">
+		<p><strong><?php esc_html_e( 'Web Stories plugin could not be initialized.', 'web-stories' ); ?></strong></p>
+		<ul>
+			<?php
+			foreach ( array_keys( $_error->errors ) as $error_code ) {
+				$message = $_error->get_error_message( $error_code );
+				printf( '<li>%s</li>', wp_kses( $message, [ 'code' => [] ] ) );
+			}
+			?>
+		</ul>
+	</div>
+	<?php
 }
 
-if ( ! class_exists( '\Google\Web_Stories\Plugin' ) ) {
-	// In CLI context, existence of the JS files is not required.
-	if ( ( defined( 'WP_CLI' ) && WP_CLI ) || 'true' === getenv( 'CI' ) || 'cli' === PHP_SAPI ) {
-		$heading = esc_html__( 'Web Stories plugin could not be initialized.', 'web-stories' );
-		$body    = sprintf(
-			/* translators: %s: build commands. */
-			esc_html__( 'You appear to be running an incomplete version of the plugin. Please run %s to finish installation.', 'web-stories' ),
-			'`composer install && npm install && npm run build`'
-		);
+add_action( 'admin_notices', __NAMESPACE__ . '\_print_missing_build_admin_notice' );
 
+if ( ( defined( 'WP_CLI' ) && WP_CLI ) || 'true' === getenv( 'CI' ) || 'cli' === PHP_SAPI ) {
+	$web_stories_compatibility->set_required_files(
+		[
+			WEBSTORIES_PLUGIN_DIR_PATH . '/third-party/vendor/scoper-autoload.php',
+			WEBSTORIES_PLUGIN_DIR_PATH . '/includes/vendor/autoload.php',
+		]
+	);
+	$web_stories_compatibility->run_checks();
+	$_error = $web_stories_compatibility->get_error();
+	if ( $_error->errors ) {
+		$heading = esc_html__( 'Web Stories plugin could not be initialized.', 'web-stories' );
 		if ( class_exists( '\WP_CLI' ) ) {
-			\WP_CLI::warning( "$heading\n$body" );
+			\WP_CLI::warning( "$heading" );
 		} else {
-			echo "$heading\n$body\n"; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			echo "$heading\n"; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
 		}
+		foreach ( array_keys( $_error->errors ) as $error_code ) {
+			$message = $_error->get_error_message( $error_code );
+			$body    = htmlspecialchars_decode( wp_strip_all_tags( $message ) );
+			if ( class_exists( '\WP_CLI' ) ) {
+				\WP_CLI::line( $body );
+			} else {
+				echo "$body\n"; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+			}
+		}
+
+		return;
 	}
+}
 
+if ( ! $web_stories_compatibility->check_php_built() ) {
 	// However, we still need to stop further execution.
 	return;
 }
+/**
+ * Run logic to setup a new site with web stories.
+ *
+ * @return void
```

Comment:
```suggestion
 *
 * @since 1.2.0
 *
 * @return void
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/4930/commits/82a9510c54d331e7b98217ededcf23dc6d468371